// idsk-react/dist/index.js
function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var idsk = require('@id-sk/frontend');
var idsk__default = _interopDefault(idsk);
var React = _interopDefault(require('react'));

function ErrorAlert(_ref) {
  var message = _ref.message,
      description = _ref.description;
  return React.createElement("div", {
    className: "govuk-error-summary",
    "aria-labelledby": "error-summary-title",
    role: "alert",
    "data-module": "govuk-error-summary"
  }, React.createElement("h2", {
    className: "govuk-error-summary__title",
    id: "error-summary-title"
  }, message), description && React.createElement("div", {
    className: "govuk-error-summary__body"
  }, React.createElement("p", null, description)));
}

function SuccessAlert(_ref) {
  var message = _ref.message;
  return React.createElement("div", {
    className: "govuk-inset-text"
  }, message);
}

function Alert(_ref) {
  var message = _ref.message,
      _ref$type = _ref.type,
      type = _ref$type === void 0 ? "error" : _ref$type,
      description = _ref.description,
      _ref$visible = _ref.visible,
      visible = _ref$visible === void 0 ? false : _ref$visible;

  function renderAlert() {
    switch (type) {
      case "error":
        return React.createElement(ErrorAlert, {
          message: message,
          description: description
        });

      case "success":
        return React.createElement(SuccessAlert, {
          message: message
        });

      default:
        return React.createElement(SuccessAlert, {
          message: message
        });
    }
  }

  return visible ? renderAlert() : null;
}

function Button(_ref) {
  var children = _ref.children,
      className = _ref.className,
      _ref$disabled = _ref.disabled,
      disabled = _ref$disabled === void 0 ? false : _ref$disabled,
      id = _ref.id,
      onClick = _ref.onClick,
      _ref$type = _ref.type,
      type = _ref$type === void 0 ? "submit" : _ref$type;
  return React.createElement("button", {
    "aria-disabled": disabled,
    className: "idsk-button " + (disabled ? "idsk-button--disabled" : "") + " " + (className || ""),
    "data-module": "idsk-button",
    disabled: disabled,
    id: id,
    onClick: onClick,
    type: type
  }, children);
}

function Container(_ref) {
  var children = _ref.children,
      className = _ref.className,
      _ref$main = _ref.main,
      main = _ref$main === void 0 ? false : _ref$main;
  return main ? React.createElement("div", {
    className: "govuk-width-container " + (className || "")
  }, React.createElement("main", {
    id: "main-content",
    className: "govuk-main-wrapper govuk-main-wrapper--auto-spacing"
  }, children)) : React.createElement("div", {
    className: "govuk-width-container " + className
  }, children);
}

function FooterLogo() {
  return React.createElement("a", {
    className: "idsk-footer__licence-logo idsk-footer__link",
    href: "https://www.mirri.gov.sk/"
  }, React.createElement("svg", {
    version: "1.1",
    id: "Layer_1",
    xmlns: "http://www.w3.org/2000/svg",
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    x: "0px",
    y: "0px",
    viewBox: "0 0 553.9584961 126.9964905",
    xmlSpace: "preserve"
  }, React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "102.7337875,19.5282993 102.7337875,6.5913 96.0428925,19.5282993 93.400795,19.5282993 \r\n\t86.907692,6.8133998 86.907692,19.5282993 83.9447937,19.5282993 83.9447937,0.1234909 87.1546936,0.1234909 94.8075943,15.6522999 \r\n\t102.7337875,0.1234909 105.6957932,0.1234909 105.6957932,19.5282993 "
  }), React.createElement("rect", {
    x: "112.1151886",
    y: "0.1234909",
    style: {
      fill: "#004c96"
    },
    width: "2.9619",
    height: "19.404808"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "140.7290039,0.123 140.7290039,19.5277996 138.3329926,19.5277996 124.4580002,5.0848999 \r\n\t124.4580002,19.5277996 121.4960938,19.5277996 121.4960938,0.123 123.9150009,0.123 137.7660828,14.5913 137.7660828,0.123 "
  }), React.createElement("rect", {
    x: "147.1459961",
    y: "0.1234909",
    style: {
      fill: "#004c96"
    },
    width: "2.96279",
    height: "19.404808"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M156.207489,14.9853001c0.8641968,0.9877996,2.7651062,2.0741901,4.3197937,2.0741901\r\n\tc1.8032074,0,3.1113129-1.0370903,3.1113129-2.8138905c0-3.3584003-8.0980988-2.8892002-8.0980988-8.8155003\r\n\tc0-3.4559999,2.8641968-5.5537,5.6791992-5.5537c1.8265991,0,3.2098999,0.667,3.973587,1.3329999V4.9863\r\n\tc-0.6908875-1.0122001-2.1718903-2.3701-3.9496918-2.3701c-1.6542969,0-2.7163086,1.1112909-2.7163086,2.6908998\r\n\tc0,3.6293912,8.1724091,3.0121999,8.1724091,8.8143902c0,3.6299095-3.0362091,5.7530003-6.2700958,5.7530003\r\n\tc-1.8032074,0-3.3090057-0.6923904-4.2221069-1.5063915V14.9853001z"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "168.0570831,0.123 183.6605835,0.123 183.6605835,2.8880999 177.3402863,2.8880999 \r\n\t177.3402863,19.5277996 174.3773956,19.5277996 174.3773956,2.8880999 168.0570831,2.8880999 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "195.8070831,19.5282993 186.7464905,19.5282993 186.7464905,0.1234909 195.8070831,0.1234909 \r\n\t195.8070831,2.8871999 189.7093964,2.8871999 189.7093964,8.4433002 194.0296936,8.4433002 194.0296936,11.1835995 \r\n\t189.7093964,11.1835995 189.7093964,16.7884903 195.8070831,16.7884903 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M206.9906921,12.1219997c-0.5429993,0.0733004-1.1103973,0.1231003-1.7031097,0.1231003h-1.0126953v7.2831993\r\n\th-2.9628906V0.123h4.0741882c4.4687042,0,7.3818207,1.8271999,7.3818207,6.0492997\r\n\tc0,2.2460999-1.0615234,3.8999-2.7646179,4.9130998l7.6777039,8.4428997h-4.1728058L206.9906921,12.1219997z M204.2748871,9.5292997\r\n\th1.1356964c2.5674133,0,4.1728973-1.0611,4.1728973-3.4073c0-2.2455997-1.5556946-3.2347999-4.0986938-3.2347999h-1.2098999\r\n\tV9.5292997z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M219.0873871,14.9853001c0.8641968,0.9877996,2.7646179,2.0741901,4.3202972,2.0741901\r\n\tc1.8027039,0,3.1113129-1.0370903,3.1113129-2.8138905c0-3.3584003-8.0986023-2.8892002-8.0986023-8.8155003\r\n\tc0-3.4559999,2.8641968-5.5537,5.6786957-5.5537c1.8271027,0,3.2089996,0.667,3.9745941,1.3329999V4.9863\r\n\tc-0.691391-1.0122001-2.1728973-2.3701-3.9501953-2.3701c-1.6542969,0-2.7158051,1.1112909-2.7158051,2.6908998\r\n\tc0,3.6293912,8.1717987,3.0121999,8.1717987,8.8143902c0,3.6299095-3.0361023,5.7530003-6.2704773,5.7530003\r\n\tc-1.8027191,0-3.3086243-0.6923904-4.2216187-1.5063915V14.9853001z"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "230.8138885,0.123 246.4164886,0.123 246.4164886,2.8880999 240.0970917,2.8880999 \r\n\t240.0970917,19.5277996 237.1341858,19.5277996 237.1341858,2.8880999 230.8138885,2.8880999 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "256.0697937,16.1210995 261.8714905,0.123 265.0560913,0.123 257.5257874,19.5277996 \r\n\t254.4154816,19.5277996 247.4037933,0.123 250.5873871,0.123 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M277.2269897,0c5.7030945,0.0493,9.7763977,3.6782,9.8993835,9.5292997\r\n\tc0.0996399,5.7279997-3.9989929,10.2708998-10.0477905,10.3685999c-6.8876953-0.0737-9.8505859-5.3569002-9.7763977-9.9727001\r\n\tC267.2767944,5.9252,270.2640991,0.0239,277.2269897,0 M277.1273804,17.1581993\r\n\tc4.3457031-0.0742989,7.0117188-3.3832998,6.9385071-7.5049C283.9662781,5.4301,281.3246765,2.7651,277.2269897,2.7402\r\n\tc-5.0126953,0.0249-6.8641968,4.3197999-6.8388977,7.1849995C270.3128967,13.2582998,272.1654968,17.1078987,277.1273804,17.1581993\r\n\t"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M83.6244888,115.9847946c0.8642044,0.9883041,2.7651062,2.0742035,4.3198013,2.0742035\r\n\tc1.8031998,0,3.1123047-1.0371017,3.1123047-2.8143997c0-3.357399-8.0991058-2.8887024-8.0991058-8.8144989\r\n\tc0-3.4550018,2.8642044-5.5547028,5.6791992-5.5547028c1.8266068,0,3.2089996,0.6679993,3.9736023,1.3339996v3.7773972\r\n\tc-0.6909027-1.0116959-2.1727982-2.3710938-3.9496994-2.3710938c-1.6542969,0-2.716301,1.1112976-2.716301,2.6913986\r\n\tc0,3.6288986,8.1724014,3.0116959,8.1724014,8.8143921c0,3.6289063-3.0362015,5.7519989-6.2701035,5.7519989\r\n\tc-1.8031998,0-3.3089981-0.691391-4.2220993-1.5058899V115.9847946z"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "109.2992935,120.5277939 99.0541916,120.5277939 99.0541916,101.1234894 102.0160904,101.1234894 \r\n\t102.0160904,117.7875977 109.2992935,117.7875977 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M120.9042892,101.0003967c5.7031021,0.0489044,9.775795,3.677803,9.8989029,9.5292969\r\n\tc0.0991058,5.7266006-3.9990005,10.2695999-10.047905,10.3692017c-6.8881989-0.0741959-9.8509979-5.3574066-9.7762909-9.9746017\r\n\tC110.9530945,106.9242935,113.941391,101.0238953,120.9042892,101.0003967 M120.8041916,118.1585999\r\n\tc4.3460999-0.0742035,7.0121994-3.3828049,6.9369965-7.505806c-0.0981903-4.2235947-2.7397995-6.8896942-6.8368988-6.9120941\r\n\tc-5.012291,0.0223999-6.8652878,4.3202972-6.8388977,7.1835938\r\n\tC113.9901886,114.2583008,115.8421936,118.1078949,120.8041916,118.1585999"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "141.2220917,117.1214905 147.0238953,101.1234894 150.2079926,101.1234894 142.6781921,120.5277939 \r\n\t139.5677948,120.5277939 132.5560913,101.1234894 135.7411957,101.1234894 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "162.8480835,120.5277939 153.7870941,120.5277939 153.7870941,101.1234894 162.8480835,101.1234894 \r\n\t162.8480835,103.8871994 156.75,103.8871994 156.75,109.4437943 161.0711823,109.4437943 161.0711823,112.1839981 \r\n\t156.75,112.1839981 156.75,117.7875977 162.8480835,117.7875977 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "187.3392944,101.1234894 187.3392944,120.5277939 184.9427948,120.5277939 171.0692902,106.0843964 \r\n\t171.0692902,120.5277939 168.1073914,120.5277939 168.1073914,101.1234894 170.5252838,101.1234894 184.3763885,115.5902939 \r\n\t184.3763885,101.1234894 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M192.9418945,115.9847946c0.8641968,0.9883041,2.7645874,2.0742035,4.3202972,2.0742035\r\n\tc1.8027039,0,3.1102905-1.0371017,3.1102905-2.8143997c0-3.357399-8.0975952-2.8887024-8.0975952-8.8144989\r\n\tc0-3.4550018,2.8641968-5.5547028,5.6786957-5.5547028c1.8271027,0,3.2098999,0.6679993,3.9746094,1.3339996v3.7773972\r\n\tc-0.6914063-1.0116959-2.1728973-2.3710938-3.9501953-2.3710938c-1.6543121,0-2.7158051,1.1112976-2.7158051,2.6913986\r\n\tc0,3.6288986,8.171814,3.0116959,8.171814,8.8143921c0,3.6289063-3.0361176,5.7519989-6.2705231,5.7519989\r\n\tc-1.8026886,0-3.3085938-0.691391-4.2215881-1.5058899V115.9847946z"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "211.3343811,101.1234894 211.3343811,110.6781998 219.8031921,101.1234894 223.7533875,101.1234894 \r\n\t214.7660828,110.8753967 223.679184,120.5277939 219.679184,120.5277939 211.3343811,111.2709961 211.3343811,120.5277939 \r\n\t208.3724823,120.5277939 208.3724823,101.1234894 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "236.6390839,120.5277939 227.579483,120.5277939 227.579483,101.1234894 236.6390839,101.1234894 \r\n\t236.6390839,103.8871994 230.5414886,103.8871994 230.5414886,109.4437943 234.8617859,109.4437943 234.8617859,112.1839981 \r\n\t230.5414886,112.1839981 230.5414886,117.7875977 236.6390839,117.7875977 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M239.9720917,124.7738953c1.7276001-1.4794998,2.5673981-3.5546951,2.5917969-5.9981003v-17.6523056h2.9873047\r\n\tv17.6269073c0,3.4804993-1.308609,6.2949982-3.6787109,8.2460938L239.9720917,124.7738953z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M266.0423889,113.1214905c-0.5429077,0.0743103-1.1112976,0.1231079-1.7030945,0.1231079h-1.0126953v7.2831955\r\n\th-2.9619141v-19.4043045h4.0732117c4.4687805,0,7.381897,1.82621,7.381897,6.0488052\r\n\tc0,2.2461014-1.0614929,3.9004059-2.7655945,4.9121017l7.6786804,8.4433975h-4.1728821L266.0423889,113.1214905z\r\n\t M263.3265991,110.5296936h1.1357727c2.5673218,0,4.172821-1.0625,4.172821-3.4082031\r\n\tc0-2.2460938-1.5556946-3.2342911-4.0986023-3.2342911h-1.2099915V110.5296936z"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "287.6440125,120.5277939 278.5834961,120.5277939 278.5834961,101.1234894 287.6440125,101.1234894 \r\n\t287.6440125,103.8871994 281.5462952,103.8871994 281.5462952,109.4437943 285.8666992,109.4437943 285.8666992,112.1839981 \r\n\t281.5462952,112.1839981 281.5462952,117.7875977 287.6440125,117.7875977 "
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M295.6175842,113.2445984v7.2831955h-2.9618835v-19.4043045h4.0733032\r\n\tc4.468689,0,7.3817749,1.82621,7.3817749,6.0488052c0,3.9258041-3.3085938,6.0723038-7.4804993,6.0723038H295.6175842z\r\n\t M295.6175842,110.5296936h1.1358032c2.5674133,0,4.1727905-1.0625,4.1727905-3.4082031\r\n\tc0-2.2460938-1.555603-3.2342911-4.0986023-3.2342911h-1.2099915V110.5296936z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M310.5296936,101.1234894v9.2832108c0,3.9979935,1.8017883,7.8007889,6.2705994,7.8007889\r\n\tc3.7030945,0,6.2460022-2.9364929,6.2460022-7.7773895v-9.3066101h2.9619751v9.2832108\r\n\tc0,6.294899-3.8007813,10.4921951-9.2079773,10.4921951c-6.4190063,0-9.2333984-4.3711014-9.2333984-10.4687958v-9.3066101\r\n\tH310.5296936z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M331.4408875,120.5277939v-19.4043045h2.7645874c4.3448181,0,7.6533203,1.3574066,7.6533203,5.2832108\r\n\tc0,1.9257889-1.0371094,3.0370941-2.8388062,3.5800934c2.7900085,0.517601,4.6416016,2.023407,4.6416016,4.8633041\r\n\tc0,4.7656021-4.5674133,5.6776962-8.5429993,5.6776962H331.4408875z M335.2181702,109.1468964\r\n\tc2.295929,0,3.6289063-0.7910004,3.6289063-2.5917969c0-2.1972046-2.1727905-2.6913986-4.0253906-2.6913986h-0.4189148v5.2831955\r\n\tH335.2181702z M335.5882874,111.8871994h-1.1855164v5.9003983h1.0869141c3.085907,0,5.085907-0.3945999,5.085907-2.8887024\r\n\tC340.575592,112.4046936,338.4515991,111.8871994,335.5882874,111.8871994"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "358.8431702,120.5277939 348.5970764,120.5277939 348.5970764,101.1234894 351.5599976,101.1234894 \r\n\t351.5599976,117.7875977 358.8431702,117.7875977 "
  }), React.createElement("rect", {
    x: "362.2260132",
    y: "101.1234894",
    style: {
      fill: "#004c96"
    },
    width: "2.9628844",
    height: "19.4043064"
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "374.8168945,101.1234894 374.8168945,110.6781998 383.2855835,101.1234894 387.2357788,101.1234894 \r\n\t378.2484741,110.8753967 387.1615906,120.5277939 383.1615906,120.5277939 374.8168945,111.2709961 374.8168945,120.5277939 \r\n\t371.8548889,120.5277939 371.8548889,101.1234894 "
  }), React.createElement("polygon", {
    style: {
      fill: "#004c96"
    },
    points: "397.5804749,120.5277939 394.6175842,120.5277939 394.6175842,112.8246994 388.2230835,101.1234894 \r\n\t391.6302795,101.1234894 396.2718811,109.8871994 400.715271,101.1234894 404.1224976,101.1234894 397.5804749,112.8753967 "
  }), React.createElement("rect", {
    x: "83.9150009",
    y: "28.2684917",
    style: {
      fill: "#d4d5d5"
    },
    width: "67.3280869",
    height: "2.9629083"
  }), React.createElement("rect", {
    x: "151.2118835",
    y: "28.2684917",
    style: {
      fill: "#004c96"
    },
    width: "67.328598",
    height: "2.9629083"
  }), React.createElement("rect", {
    x: "218.5082855",
    y: "28.2684917",
    style: {
      fill: "#e50b14"
    },
    width: "67.3281021",
    height: "2.9629083"
  }), React.createElement("path", {
    style: {
      fill: "#e50b14"
    },
    d: "M32.2621918,80.8080978C18.989193,74.4125977,0,61.7220993,0,36.6170998\r\n\tC0,11.5130997,1.2001938,0.0961,1.2001938,0.0961H63.324192c0,0,1.2030983,11.4169998,1.2030983,36.5209999\r\n\tC64.5272903,61.7220993,45.5379982,74.4125977,32.2621918,80.8080978"
  }), React.createElement("g", null, React.createElement("defs", null, React.createElement("path", {
    id: "SVGID_1_",
    d: "M32.2621918,0.0957H1.2010937C1.2010937,0.0957,0,11.5127001,0,36.6175995\r\n\t\t\tc0,25.1035004,18.989193,37.7958908,32.2621918,44.1904984c13.2758064-6.3946075,32.2636032-19.086998,32.2636032-44.1904984\r\n\t\t\tc0-25.1049004-1.2006035-36.5219002-1.2006035-36.5219002H32.2621918z"
  })), React.createElement("clipPath", {
    id: "SVGID_2_"
  }, React.createElement("use", {
    xlinkHref: "#SVGID_1_",
    style: {
      overflow: "visible"
    }
  })), React.createElement("path", {
    style: {
      clipPath: "url(#SVGID_2_)",
      fill: "#ffffff"
    },
    d: "M35.1024933,30.0340996c3.5942001,0.0601006,10.6068993,0.2001991,16.8500977-1.8910999\r\n\t\tc0,0-0.1640968,2.2383003-0.1640968,4.8398991c0,2.6020012,0.1640968,4.8363991,0.1640968,4.8363991\r\n\t\tc-5.7250977-1.9149971-12.8002968-1.9550972-16.8500977-1.9042969v13.8651962h-5.6763V35.9150009\r\n\t\tc-4.0517006-0.0508003-11.1259003-0.0107002-16.8525009,1.9042969c0,0,0.165-2.2343979,0.165-4.8363991\r\n\t\tc0-2.6015987-0.165-4.8398991-0.165-4.8398991c6.2426996,2.0912991,13.2568007,1.9512005,16.8525009,1.8910999v-8.7070007\r\n\t\tc-3.2758999-0.0277996-7.9974995,0.1279011-13.3348999,1.9141006c0,0,0.1639996-2.2348995,0.1639996-4.8388996\r\n\t\tc0-2.6001005-0.1639996-4.8364-0.1639996-4.8364c5.3281002,1.7821999,10.0429001,1.9424,13.3183002,1.9139996\r\n\t\tc-0.1689014-5.5127001-1.773901-12.4637995-1.773901-12.4637995s3.3057003,0.2558,4.6269989,0.2558\r\n\t\tc1.3237,0,4.6289024-0.2558,4.6289024-0.2558s-1.6060028,6.9510994-1.7740021,12.4637995\r\n\t\tc3.2749023,0.0284004,7.9897995-0.1317997,13.3179016-1.9139996c0,0-0.1641006,2.2362995-0.1641006,4.8364\r\n\t\tc0,2.6040001,0.1641006,4.8388996,0.1641006,4.8388996c-5.3358994-1.7861996-10.0581017-1.9419003-13.3330002-1.9141006V30.0340996\r\n\t\t"
  }), React.createElement("path", {
    style: {
      clipPath: "url(#SVGID_2_)",
      fillRule: "evenodd",
      clipRule: "evenodd",
      fill: "#004c96"
    },
    d: "M31.6795921,89.0443954L4.1747937,64.5248947c0,0,3.5135999-11.3715973,10.3797989-11.3715973\r\n\t\tc5.4532003,0,7.4429007,4.3969002,7.4429007,4.3969002s3.5800991-9.2700005,10.2651978-9.2700005\r\n\t\tc6.6870003,0,10.2670021,9.2700005,10.2670021,9.2700005s1.9888-4.3969002,7.4438972-4.3969002\r\n\t\tc6.8638,0,10.3789024,11.3715973,10.3789024,11.3715973L31.6795921,89.0443954"
  })), React.createElement("g", null, React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M85.9164886,43.6400986v19.3768997h-2.9584885V43.6400986H85.9164886z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M109.0692902,43.6400986v19.3768997h-2.391098L92.8216934,48.5942001v14.4227982h-2.9585037V43.6400986\r\n\t\th2.4160995l13.8315048,14.4468002V43.6400986H109.0692902z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M119.9486923,59.6180992l5.7938995-15.9780006h3.1807022l-7.5195007,19.3768997h-3.1079025\r\n\t\tl-7.0009995-19.3768997h3.180603L119.9486923,59.6180992z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M140.0716858,63.0169983h-9.0477905V43.6400986h9.0477905v2.7616997h-6.089798v5.5474014h4.3144073v2.7372894\r\n\t\th-4.3144073v5.5957108h6.089798V63.0169983z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M142.7865906,58.4813995c0.8623047,0.9862976,2.7597046,2.0723,4.3125,2.0723\r\n\t\tc1.8016968,0,3.1094055-1.0352097,3.1094055-2.8106003c0-3.3535004-8.0870056-2.8857002-8.0870056-8.8017006\r\n\t\tc0-3.4527016,2.8583984-5.5479088,5.6699982-5.5479088c1.8242035,0,3.2041016,0.6651077,3.969696,1.3306084v3.7733917\r\n\t\tc-0.6894989-1.011692-2.1698914-2.367691-3.9442902-2.367691c-1.6534119,0-2.7128906,1.1094017-2.7128906,2.6865997\r\n\t\tc0,3.6264,8.161087,3.0078011,8.161087,8.8017006c0,3.6240005-3.0332031,5.7486-6.2637024,5.7486\r\n\t\tc-1.7998047,0-3.3016968-0.6954002-4.214798-1.5059013V58.4813995z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M154.2533875,43.6400986h15.5825043v2.7616997h-6.3106079v16.6152h-2.9588928v-16.6152h-6.3130035V43.6400986z\r\n\t\t"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M172.9955902,42.7533989l-1.5776978-1.5064011l3.550293-3.6742973l1.6524048,1.7279968L172.9955902,42.7533989\r\n\t\tz M174.7709961,43.6400986v19.3768997h-2.9585114V43.6400986H174.7709961z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M177.7552948,53.4272003c-0.023407-3.9922028,2.9589996-10.0337105,9.9101868-10.0337105\r\n\t\tc2.7383118,0,4.8105164,0.7647095,6.1660004,1.8486099v3.5014992c-1.406189-1.4048004-3.1327972-2.6138-6.1660004-2.6138\r\n\t\tc-5.003891,0-6.851593,4.4379997-6.8280945,7.2974014c0,3.1322899,1.8242035,7.1762924,6.8280945,7.1762924\r\n\t\tc2.5898132,0,4.7851105-0.914093,6.1660004-2.2197952v3.1054993c-1.531189,1.1332932-3.230484,1.8521004-6.1660004,1.8521004\r\n\t\tc-6.9511871-0.0253983-9.9101868-5.525898-9.9101868-9.889698V53.4272003z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M199.7513885,43.6400986v19.3768997h-2.9589996V43.6400986H199.7513885z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M204.8822937,42.7533989l-1.5762024-1.5064011l3.5469055-3.6742973l1.6542969,1.7279968\r\n\t\tL204.8822937,42.7533989z M206.6576843,43.6400986v19.3768997h-2.9609985V43.6400986H206.6576843z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M213.9564819,60.0116997c0.0742035,2.192791-0.3690948,4.4603996-1.75,5.6967888h-2.9570007\r\n\t\tc1.2793121-1.7050934,1.7480011-3.4550896,1.7480011-5.6967888H213.9564819z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M227.4212952,55.6239967c-0.5429077,0.0752029-1.1092987,0.1211014-1.7011108,0.1211014h-1.0117035v7.2719002\r\n\t\th-2.9570923V43.6400986h4.0664063c4.4609985,0,7.3730927,1.8232002,7.3730927,6.0419998\r\n\t\tc0,2.2416992-1.0625,3.8945007-2.7617035,4.9028015l7.6660004,8.4320984h-4.1660004L227.4212952,55.6239967z\r\n\t\t M224.7084808,53.0331993h1.1347046c2.5644989,0,4.1679993-1.0586014,4.1679993-3.4019012\r\n\t\tc0-2.2431984-1.5567017-3.2294998-4.0957031-3.2294998h-1.2070007V53.0331993z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M248.0111847,63.0169983h-9.0468903V43.6400986h9.0468903v2.7616997h-6.0878906v5.5474014h4.3144989v2.7372894\r\n\t\th-4.3144989v5.5957108h6.0878906V63.0169983z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M250.0599976,53.4272003c0-3.9922028,2.9120941-10.0337105,10.1582031-10.0337105\r\n\t\tc2.171875,0,4.5605774,0.4683075,5.9179993,1.3062096v3.5014c-1.4043274-1.1845016-3.328125-2.0713005-5.9179993-2.0713005\r\n\t\tc-5.2754059,0-7.0762024,4.4379997-7.0762024,7.2974014c0,3.1322899,1.824295,7.1762924,6.8300781,7.1762924\r\n\t\tc1.875,0,2.9590149-0.2958946,4.1934204-0.789093v-5.6719093h2.9570007v7.0537071\r\n\t\tc-1.8242188,1.2564011-4.2148132,2.1450996-7.1504211,2.1450996c-6.9550781-0.0253983-9.9120789-5.525898-9.9120789-9.889698\r\n\t\tV53.4272003z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M273.5365906,43.6400986v19.3768997h-2.9590149V43.6400986H273.5365906z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M286.433075,43.5174904c5.6953125,0.0474091,9.765625,3.6739082,9.8867188,9.5157089\r\n\t\tc0.0995789,5.7196999-3.992218,10.2572899-10.0332031,10.358799c-6.8789063-0.0760994-9.8359985-5.3540001-9.7655945-9.964798\r\n\t\tC276.4974976,49.4349976,279.4798889,43.5394897,286.433075,43.5174904z M286.3353882,60.652298\r\n\t\tc4.3399048-0.0741997,7.0019836-3.3778992,6.9296875-7.4940987c-0.0995789-4.2168007-2.7382813-6.8769989-6.8320007-6.9034004\r\n\t\tc-5.0038757,0.0264015-6.8535767,4.3129997-6.828186,7.1724014\r\n\t\tC279.5287781,56.7557983,281.3783875,60.6034927,286.3353882,60.652298z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M318.5169983,43.6400986v19.3768997h-2.3925171l-13.8574829-14.4227982v14.4227982h-2.9589233V43.6400986\r\n\t\th2.4179382l13.8320618,14.4468002V43.6400986H318.5169983z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M331.0462952,43.6400986l7.1757813,19.3768997h-3.1796875l-2.2929077-6.5825081h-6.5116882\r\n\t\tl-2.3184204,6.5825081h-3.1776733l7.4198914-19.3768997H331.0462952z M327.200592,53.6987h4.5859985l-2.2675781-6.5326996\r\n\t\tL327.200592,53.6987z M328.925293,42.7533989l-1.5782166-1.5064011l3.5527954-3.6742973l1.6504211,1.7279968L328.925293,42.7533989\r\n\t\tz"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M350.675293,63.0169983h-10.2325134V43.6400986h2.9609985v16.6421013h7.2715149V63.0169983z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M372.2787781,43.6400986v19.3768997h-2.3945923l-13.8573914-14.4227982v14.4227982h-2.9551086V43.6400986\r\n\t\th2.414093l13.8320007,14.4468002V43.6400986H372.2787781z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M385.1537781,63.0169983h-9.0469055V43.6400986h9.0469055v2.7616997h-6.0917664v5.5474014h4.3163757v2.7372894\r\n\t\th-4.3163757v5.5957108h6.0917664V63.0169983z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M400.9857788,43.6400986h2.9551086v19.3768997h-2.9551086v-8.3070984h-9.8397827v8.3070984h-2.9589844\r\n\t\tV43.6400986h2.9589844v8.3344994h9.8397827V43.6400986z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M416.8412781,43.5174904c5.6972046,0.0474091,9.7655945,3.6739082,9.8905945,9.5157089\r\n\t\tc0.0957031,5.7196999-3.9980774,10.2572899-10.039093,10.358799c-6.8789063-0.0760994-9.8339844-5.3540001-9.7597046-9.964798\r\n\t\tC406.9076843,49.4349976,409.888092,43.5394897,416.8412781,43.5174904z M416.7416687,60.652298\r\n\t\tc4.3418274-0.0741997,7.0019226-3.3778992,6.9277039-7.4940987c-0.0957031-4.2168007-2.734375-6.8769989-6.8280945-6.9034004\r\n\t\tc-5.0039063,0.0264015-6.8534851,4.3129997-6.828186,7.1724014\r\n\t\tC409.9370117,56.7557983,411.7884827,60.6034927,416.7416687,60.652298z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M442.7884827,55.6239967c-0.5429077,0.0752029-1.1131897,0.1211014-1.7030945,0.1211014h-1.0078125v7.2719002\r\n\t\th-2.960907V43.6400986h4.0664063c4.4648132,0,7.3730164,1.8232002,7.3730164,6.0419998\r\n\t\tc0,2.2416992-1.0605164,3.8945007-2.7617188,4.9028015l7.6679993,8.4320984h-4.1620789L442.7884827,55.6239967z\r\n\t\t M440.0775757,53.0331993h1.1309204c2.5663757,0,4.1678772-1.0586014,4.1678772-3.4019012\r\n\t\tc0-2.2431984-1.5546875-3.2294998-4.0917969-3.2294998h-1.2070007V53.0331993z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M463.4017944,43.5174904c5.695282,0.0474091,9.7655945,3.6739082,9.8866882,9.5157089\r\n\t\tc0.0976868,5.7196999-3.9941101,10.2572899-10.0371094,10.358799c-6.8768921-0.0760994-9.8320007-5.3540001-9.7616882-9.964798\r\n\t\tC453.4662781,49.4349976,456.450592,43.5394897,463.4017944,43.5174904z M463.3021851,60.652298\r\n\t\tc4.3418274-0.0741997,7.0039063-3.3778992,6.9296875-7.4940987c-0.0995789-4.2168007-2.7362976-6.8769989-6.8300781-6.9034004\r\n\t\tc-5.0019226,0.0264015-6.8515015,4.3129997-6.8262024,7.1724014\r\n\t\tC456.4994812,56.7557983,458.3490906,60.6034927,463.3021851,60.652298z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M475.3158875,43.6400986h12.6268921v2.1929016l-8.9784851,14.4491997h8.9042969v2.7347984h-12.5780945\r\n\t\tv-2.1917992l8.8535156-14.4234009h-8.828125V43.6400986z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M497.8294678,59.6180992l5.7911072-15.9780006h3.1835938l-7.5195923,19.3768997h-3.1034851\r\n\t\tl-7.0057983-19.3768997h3.1815796L497.8294678,59.6180992z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M517.9760132,43.5174904c5.6972656,0.0474091,9.765686,3.6739082,9.890686,9.5157089\r\n\t\tc0.0957031,5.7196999-3.9960938,10.2572899-10.0352173,10.358799c-6.8789063-0.0760994-9.8378906-5.3540001-9.7637024-9.964798\r\n\t\tC508.0423889,49.4349976,511.0248718,43.5394897,517.9760132,43.5174904z M517.8783569,60.652298\r\n\t\tc4.3398438-0.0741997,7.0019531-3.3778992,6.9277344-7.4940987c-0.0957031-4.2168007-2.734436-6.8769989-6.8300781-6.9034004\r\n\t\tc-5.0039063,0.0264015-6.851532,4.3129997-6.828125,7.1724014\r\n\t\tC511.0736694,56.7557983,512.9232788,60.6034927,517.8783569,60.652298z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M528.6831055,67.259201c1.7265015-1.4804001,2.5643921-3.5488014,2.5897827-5.9917107V43.6400986h2.9824219\r\n\t\tv17.6020012c0,3.4799995-1.3047485,6.2905998-3.6738281,8.2358971L528.6831055,67.259201z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M546.7846069,43.6400986l7.1738892,19.3768997h-3.1817017l-2.2930298-6.5825081h-6.5097046\r\n\t\tl-2.3164063,6.5825081h-3.1816406l7.4238892-19.3768997H546.7846069z M542.9370117,53.6987h4.5858765l-2.2695313-6.5326996\r\n\t\tL542.9370117,53.6987z"
  })), React.createElement("g", null, React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M92.5677948,72.3119965l7.2060928,19.464798h-3.1938858l-2.3042068-6.6113052h-6.5366058l-2.3280945,6.6113052\r\n\t\th-3.1943054l7.4536057-19.464798H92.5677948z M88.7044907,82.4154892h4.6060028l-2.2778015-6.5634918L88.7044907,82.4154892z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M112.4085922,72.3119965v19.464798h-2.9720993v-19.464798H112.4085922z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M135.6625977,72.3119965v19.464798h-2.4019012L119.343689,77.2894897v14.4873047h-2.9721985v-19.464798\r\n\t\th2.4272995l13.8931046,14.510704v-14.510704H135.6625977z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M139.5067902,91.7767944v-19.464798h9.0879059v2.7733994h-6.1162109v5.6220932h4.3091125v2.7490005h-4.3091125\r\n\t\tv8.3203049H139.5067902z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M159.6430054,72.1888962c5.7201843,0.0498047,9.8056793,3.6884995,9.9296875,9.5575943\r\n\t\tc0.0986023,5.746109-4.012207,10.3018036-10.078598,10.4024048c-6.9092102-0.0761948-9.8813019-5.3740005-9.8070984-10.0058975\r\n\t\tC149.6620941,78.1312943,152.6585846,72.213295,159.6430054,72.1888962z M159.5423889,89.3988953\r\n\t\tc4.3598938-0.0741959,7.0337067-3.392601,6.9584961-7.5274048c-0.0986023-4.2333908-2.7480011-6.9090958-6.8578796-6.934494\r\n\t\tc-5.0278168,0.0253983-6.8857117,4.3339996-6.860321,7.2060013\r\n\t\tC152.7088928,85.4847946,154.5658875,89.3500977,159.5423889,89.3988953z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M178.2709961,84.3480988c-0.5469055,0.0741959-1.1133118,0.125-1.7089996,0.125h-1.0157013v7.3036957\r\n\t\th-2.9730988v-19.464798h4.0883942c4.4804077,0,7.402298,1.8310013,7.402298,6.0673981\r\n\t\tc0,2.2539063-1.0643921,3.913002-2.773407,4.9277039l7.7012024,8.469696h-4.1835938L178.2709961,84.3480988z\r\n\t\t M175.5462952,81.7464905h1.1407013c2.5741882,0,4.1835938-1.0633926,4.1835938-3.4160004\r\n\t\tc0-2.2548904-1.5605927-3.2450943-4.1114044-3.2450943h-1.2128906V81.7464905z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M208.8314819,91.7767944v-12.976593l-6.7089996,12.976593h-2.6503906l-6.5135956-12.7538986v12.7538986\r\n\t\th-2.9707031v-19.464798h3.2166901l7.6778107,15.5771027l7.9491882-15.5771027h2.972702v19.464798H208.8314819z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M224.388092,72.3119965l7.2050934,19.464798h-3.191391l-2.3047028-6.6113052h-6.5390015l-2.3262024,6.6113052\r\n\t\th-3.1952972l7.4530945-19.464798H224.388092z M220.5248871,82.4154892h4.6074066l-2.2792969-6.5634918L220.5248871,82.4154892z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M231.8451843,72.3119965h15.652298v2.7733994h-6.339798v16.6913986h-2.9726868V75.0853958h-6.3398132\r\n\t\tV72.3119965z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M252.4525909,72.3119965v19.464798h-2.9707031v-19.464798H252.4525909z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M255.4525909,72.3119965h12.6776886v2.2041016l-9.0155945,14.5106964h8.9395142v2.75h-12.6270142v-2.2040939\r\n\t\tl8.8887024-14.4873047h-8.8632965V72.3119965z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M279.7220764,72.3119965l7.2051086,19.464798h-3.1933899l-2.3027039-6.6113052h-6.5370789l-2.328125,6.6113052\r\n\t\th-3.1933899l7.4530945-19.464798H279.7220764z M275.8587952,82.4154892h4.6054993l-2.2772827-6.5634918L275.8587952,82.4154892z\r\n\t\t M277.5932007,71.4212952l-1.5840149-1.5116959l3.5645142-3.6903992l1.6581726,1.7352905L277.5932007,71.4212952z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M288.1947937,82.1429977c-0.0234985-4.0117035,2.9707031-10.0791016,9.9550781-10.0791016\r\n\t\tc2.75,0,4.8300171,0.7685928,6.1894226,1.8564987v3.5166016c-1.4100952-1.4112015-3.1426086-2.6221008-6.1894226-2.6221008\r\n\t\tc-5.0273743,0-6.8828735,4.4561005-6.859375,7.3281021c0,3.1454926,1.8320007,7.2070999,6.859375,7.2070999\r\n\t\tc2.6015015,0,4.8046265-0.9169998,6.1894226-2.2294998v3.1201019c-1.5332031,1.1386948-3.242218,1.857399-6.1894226,1.857399\r\n\t\tc-6.984375-0.0253983-9.9550781-5.5468979-9.9550781-9.9307022V82.1429977z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M310.2884827,72.3119965v19.464798h-2.9725952v-19.464798H310.2884827z"
  }), React.createElement("path", {
    style: {
      fill: "#004c96"
    },
    d: "M323.2162781,91.7767944h-9.0878906v-19.464798h9.0878906v2.7733994h-6.1171875v5.5723038h4.3339844v2.7490005\r\n\t\th-4.3339844v5.6200943h6.1171875V91.7767944z"
  }))));
}

function FooterMZLogo() {
  return React.createElement("a", {
    className: "idsk-footer__licence-logo idsk-footer__link",
    href: "https://www.health.gov.sk/"
  }, React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 404.12 98.37"
  }, React.createElement("defs", null, React.createElement("clipPath", {
    id: "a",
    transform: "translate(-247.36 -236.45)"
  }, React.createElement("rect", {
    style: {
      fill: "none"
    },
    x: "259",
    y: "239",
    width: "41",
    height: "48"
  })), React.createElement("clipPath", {
    id: "b",
    transform: "translate(-247.36 -236.45)"
  }, React.createElement("path", {
    style: {
      fill: "none"
    },
    d: "M279.62,236.67H248.56s-1.2,11.42-1.2,36.52,19,37.8,32.26,44.19c13.27-6.39,32.26-19.08,32.26-44.19s-1.2-36.52-1.2-36.52Z"
  })), React.createElement("clipPath", {
    id: "c",
    transform: "translate(-247.36 -236.45)"
  }, React.createElement("rect", {
    style: {
      fill: "none"
    },
    x: "251",
    y: "284",
    width: "57",
    height: "34"
  }))), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M350.09,256.1V243.16L343.4,256.1h-2.64l-6.49-12.71V256.1h-3V236.7h3.21l7.65,15.53,7.93-15.53h3v19.4Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M362.43,256.1h-3V236.7h3Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M388.09,236.7v19.4h-2.4l-13.88-14.44V256.1h-3V236.7h2.42l13.85,14.46V236.7Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M397.46,256.1h-3V236.7h3Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M403.56,251.56a6.66,6.66,0,0,0,4.32,2.07c1.81,0,3.12-1,3.12-2.81,0-3.36-8.1-2.89-8.1-8.82a5.56,5.56,0,0,1,5.67-5.55,6.14,6.14,0,0,1,4,1.33v3.78a5.2,5.2,0,0,0-3.95-2.37,2.56,2.56,0,0,0-2.72,2.69c0,3.63,8.17,3,8.17,8.82,0,3.62-3,5.75-6.26,5.75a6.48,6.48,0,0,1-4.23-1.51Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M415.41,236.7H431v2.76H424.7V256.1h-3V239.46h-6.32Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M443.16,256.1H434.1V236.7h9.06v2.76h-6.09V245h4.32v2.74h-4.32v5.6h6.09Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M451.63,246.1h1.14c2.56,0,4.17-1.06,4.17-3.4s-1.56-3.24-4.1-3.24h-1.21Zm2.72,2.6a13.5,13.5,0,0,1-1.71.12h-1v7.28h-3V236.7h4.07c4.47,0,7.39,1.82,7.39,6.05a5.48,5.48,0,0,1-2.77,4.91L465,256.1h-4.18Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M466.45,251.56a6.62,6.62,0,0,0,4.31,2.07c1.81,0,3.12-1,3.12-2.81,0-3.36-8.1-2.89-8.1-8.82a5.56,5.56,0,0,1,5.68-5.55,6.16,6.16,0,0,1,4,1.33v3.78a5.2,5.2,0,0,0-3.95-2.37,2.56,2.56,0,0,0-2.72,2.69c0,3.63,8.17,3,8.17,8.82,0,3.62-3,5.75-6.27,5.75a6.42,6.42,0,0,1-4.21-1.51Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M478.17,236.7h15.6v2.76h-6.32V256.1h-3V239.46h-6.32Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M503.43,252.7l5.8-16h3.18l-7.53,19.4h-3.11l-7-19.4H498Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M524.48,253.73c4.35-.07,7-3.38,6.94-7.5s-2.74-6.89-6.84-6.92c-5,0-6.86,4.32-6.83,7.19-.08,3.33,1.77,7.18,6.73,7.23m.1-17.16c5.71.05,9.78,3.68,9.9,9.53a9.9,9.9,0,0,1-10.05,10.37c-6.88-.07-9.84-5.36-9.77-10a9.85,9.85,0,0,1,9.92-9.93",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M330.34,280.2H343v2.19l-9,14.47h8.92v2.74h-12.6v-2.19L339.18,283h-8.84Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M350.83,296.86H352c5.62,0,8.88-1.8,8.88-7.06,0-5.06-3.38-6.84-9-6.84h-1.11Zm-3,2.74V280.2h3.45c7.93,0,12.69,2.69,12.69,9.77,0,6.72-4.76,9.63-12.64,9.63Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M371.17,289.6h1.14c2.57,0,4.17-1.06,4.17-3.4s-1.55-3.24-4.1-3.24h-1.21Zm2.72,2.6a13.37,13.37,0,0,1-1.71.12h-1v7.28h-3V280.2h4.07c4.47,0,7.38,1.82,7.38,6.05a5.45,5.45,0,0,1-2.76,4.91l7.68,8.44h-4.17Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M391.05,290.27h4.59l-2.27-6.54Zm3.85-10.07,7.18,19.4H398.9L396.6,293h-6.52l-2.32,6.59h-3.18L392,280.2Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M412.23,296.2l5.8-16h3.18l-7.53,19.4h-3.11l-7-19.4h3.19Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M433.29,297.23c4.34-.07,7-3.38,6.93-7.5s-2.74-6.89-6.83-6.92c-5,0-6.87,4.32-6.84,7.19-.08,3.33,1.77,7.18,6.74,7.23m.1-17.16c5.7.05,9.77,3.68,9.9,9.53A9.9,9.9,0,0,1,433.24,300c-6.89-.07-9.85-5.36-9.78-10a9.85,9.85,0,0,1,9.93-9.93",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M445.53,280.2h15.61V283h-6.33V299.6h-3V283h-6.32Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M484.32,280.2v19.4h-2.4l-13.87-14.44V299.6h-3V280.2h2.41l13.86,14.46V280.2Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M493.7,299.6h-3V280.2h3Zm-1.78-20.29-1.58-1.51,3.55-3.68,1.66,1.73Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M499.16,290c0-4,3-10.05,9.92-10.05a9.75,9.75,0,0,1,6.17,1.85v3.51a8,8,0,0,0-6.17-2.62c-5,0-6.87,4.44-6.84,7.31,0,3.13,1.83,7.18,6.84,7.18a9,9,0,0,0,6.17-2.22v3.11a9.4,9.4,0,0,1-6.17,1.85c-7,0-9.92-5.53-9.92-9.9Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M518.21,280.2h15.61V283H527.5V299.6h-3V283h-6.32Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M544.21,296.2l5.8-16h3.19l-7.54,19.4h-3.11l-7-19.4h3.19Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M561.64,290.27h4.59L564,283.73Zm3.85-10.07,7.18,19.4h-3.18L567.2,293h-6.52l-2.32,6.59h-3.19l7.43-19.4Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M331,323.81a6.64,6.64,0,0,0,4.32,2.07,2.8,2.8,0,0,0,3.11-2.81c0-3.36-8.1-2.89-8.1-8.82A5.57,5.57,0,0,1,336,308.7a6.06,6.06,0,0,1,4,1.34v3.77a5.18,5.18,0,0,0-3.94-2.37,2.56,2.56,0,0,0-2.72,2.69c0,3.63,8.17,3,8.17,8.82,0,3.62-3,5.75-6.27,5.75a6.45,6.45,0,0,1-4.22-1.51Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M356.66,328.35H346.41V309h3v16.66h7.29Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M368.16,326c4.35-.07,7-3.38,6.94-7.5s-2.74-6.89-6.84-6.92c-5,0-6.87,4.32-6.84,7.19-.07,3.33,1.78,7.18,6.74,7.23m.1-17.16c5.7.06,9.78,3.68,9.9,9.53a9.9,9.9,0,0,1-10.05,10.37c-6.89-.07-9.85-5.36-9.77-10a9.85,9.85,0,0,1,9.92-9.93",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M388.58,325l5.8-16h3.18L390,328.35h-3.11l-7-19.4h3.19Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M410.2,328.35h-9.06V309h9.06v2.76h-6.09v5.56h4.32V320h-4.32v5.6h6.09Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M434.7,309v19.4h-2.4l-13.87-14.44v14.44h-3V309h2.42l13.85,14.46V309Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M440.3,323.81a6.62,6.62,0,0,0,4.32,2.07,2.8,2.8,0,0,0,3.11-2.81c0-3.36-8.1-2.89-8.1-8.82a5.57,5.57,0,0,1,5.68-5.55,6.11,6.11,0,0,1,4,1.34v3.77a5.24,5.24,0,0,0-4-2.37,2.56,2.56,0,0,0-2.71,2.69c0,3.63,8.17,3,8.17,8.82,0,3.62-3,5.75-6.27,5.75a6.45,6.45,0,0,1-4.22-1.51Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M458.69,309v9.55l8.47-9.55h3.95l-9,9.75,8.92,9.65h-4l-8.35-9.26v9.26h-3V309Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M484,328.35h-9.07V309H484v2.76h-6.1v5.56h4.32V320H477.9v5.6H484Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M487.33,332.6a7.73,7.73,0,0,0,2.59-6V309h3v17.62a10.27,10.27,0,0,1-3.68,8.25Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M510.68,318.35h1.14c2.57,0,4.17-1.06,4.17-3.4s-1.55-3.24-4.1-3.24h-1.21Zm2.72,2.6a13.21,13.21,0,0,1-1.7.12h-1v7.28h-3V309h4.07c4.47,0,7.39,1.82,7.39,6.05a5.48,5.48,0,0,1-2.77,4.91l7.68,8.44h-4.18Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M535,328.35h-9.06V309H535v2.76h-6.1v5.56h4.32V320H528.9v5.6H535Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M543,318.35h1.14c2.57,0,4.17-1.06,4.17-3.4s-1.55-3.24-4.1-3.24H543Zm0,2.72v7.28h-3V309h4.08c4.46,0,7.37,1.82,7.37,6.05,0,3.92-3.3,6.07-7.47,6.07Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M557.89,309v9.28c0,4,1.8,7.8,6.27,7.8,3.7,0,6.24-2.94,6.24-7.78V309h3v9.28c0,6.29-3.8,10.49-9.2,10.49-6.42,0-9.24-4.37-9.24-10.47V309Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M583,319.71h-1.19v5.9h1.08c3.09,0,5.09-.4,5.09-2.89s-2.12-3-5-3m-.38-2.74c2.3,0,3.63-.79,3.63-2.59,0-2.2-2.17-2.69-4-2.69h-.42V317Zm-3.77,11.38V309h2.76c4.35,0,7.65,1.35,7.65,5.28a3.48,3.48,0,0,1-2.83,3.58c2.79.52,4.64,2,4.64,4.86,0,4.77-4.57,5.68-8.55,5.68Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M606.2,328.35H596V309h3v16.66h7.28Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M612.55,328.35h-3V309h3Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M622.17,309v9.55l8.47-9.55h4l-9,9.75,8.91,9.65h-4l-8.35-9.26v9.26h-3V309Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M644.94,328.35h-3v-7.7L635.58,309H639l4.64,8.76,4.44-8.76h3.41l-6.54,11.75Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#d3d2d2"
    },
    d: "M398.6,267.8H331.27v-3H398.6Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#1e51a4"
    },
    d: "M465.89,267.8H398.57v-3h67.32Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#ed1c24"
    },
    d: "M533.19,267.8H465.86v-3h67.33Z",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("path", {
    style: {
      fill: "#ed1c24"
    },
    d: "M279.62,317.38c-13.28-6.39-32.26-19.08-32.26-44.19s1.19-36.52,1.19-36.52h62.13s1.2,11.42,1.2,36.52-19,37.8-32.26,44.19",
    transform: "translate(-247.36 -236.45)"
  }), React.createElement("g", {
    style: {
      clipPath: "url(#a)"
    }
  }, React.createElement("g", {
    style: {
      clipPath: "url(#b)"
    }
  }, React.createElement("path", {
    style: {
      fill: "#ffffff"
    },
    d: "M282.46,266.61c3.59.06,10.61.2,16.85-1.9,0,0-.17,2.24-.17,4.84s.17,4.84.17,4.84c-5.73-1.91-12.8-1.95-16.85-1.9v13.87h-5.68V272.49c-4-.05-11.12,0-16.85,1.9,0,0,.16-2.23.16-4.84s-.16-4.84-.16-4.84c6.24,2.1,13.26,2,16.85,1.9V257.9a39.77,39.77,0,0,0-13.33,1.92s.16-2.24.16-4.84-.16-4.84-.16-4.84a39.86,39.86,0,0,0,13.32,1.91A77.36,77.36,0,0,0,275,239.59s3.31.25,4.63.25,4.63-.25,4.63-.25a77.36,77.36,0,0,0-1.78,12.46,39.86,39.86,0,0,0,13.32-1.91s-.16,2.24-.16,4.84.16,4.84.16,4.84a39.73,39.73,0,0,0-13.33-1.92v8.71",
    transform: "translate(-247.36 -236.45)"
  }))), React.createElement("g", {
    style: {
      clipPath: "url(#c)"
    }
  }, React.createElement("g", {
    style: {
      clipPath: "url(#b)"
    }
  }, React.createElement("path", {
    style: {
      fill: "rgb(0, 76, 150)"
    },
    d: "M279,325.62,251.53,301.1s3.51-11.37,10.38-11.37a8.22,8.22,0,0,1,7.45,4.4s3.57-9.27,10.26-9.27,10.27,9.27,10.27,9.27a8.2,8.2,0,0,1,7.44-4.4c6.87,0,10.38,11.37,10.38,11.37L279,325.62",
    transform: "translate(-247.36 -236.45)"
  })))));
}

function Footer(_ref) {
  var className = _ref.className,
      _ref$logo = _ref.logo,
      logo = _ref$logo === void 0 ? "MIRI" : _ref$logo,
      labels = _ref.labels;

  function renderLogo() {
    switch (logo) {
      case "MIRI":
        return React.createElement(FooterLogo, null);

      case "MZ":
        return React.createElement(FooterMZLogo, null);

      default:
        return React.createElement(FooterLogo, null);
    }
  }

  return React.createElement("footer", {
    className: "idsk-footer " + (className || ""),
    role: "contentinfo"
  }, React.createElement("div", {
    className: "govuk-width-container"
  }, React.createElement("div", {
    className: "idsk-footer__meta"
  }, React.createElement("div", {
    className: "idsk-footer__meta-item"
  }, renderLogo()), React.createElement("div", {
    className: "idsk-footer__meta-item idsk-footer__meta-item--grow"
  }, labels && labels.map(function (label, index) {
    return React.createElement("div", {
      key: "footer-label-" + index
    }, React.createElement("span", {
      className: "idsk-footer__licence-description"
    }, label));
  }), React.createElement("h2", {
    className: "govuk-visually-hidden"
  }, "Support links")))));
}

function Grid(_ref) {
  var children = _ref.children,
      className = _ref.className,
      _ref$width = _ref.width,
      width = _ref$width === void 0 ? "full" : _ref$width;

  function renderGrid() {
    switch (width) {
      case "row":
        return React.createElement("div", {
          className: "govuk-grid-row " + (className || "")
        }, children);

      case "full":
        return React.createElement("div", {
          className: "govuk-grid-column-full " + (className || "")
        }, children);

      case "half":
        return React.createElement("div", {
          className: "govuk-grid-column-one-half " + (className || "")
        }, children);

      case "third":
        return React.createElement("div", {
          className: "govuk-grid-column-one-third " + (className || "")
        }, children);

      case "two-thirds":
        return React.createElement("div", {
          className: "govuk-grid-column-two-thirds " + (className || "")
        }, children);

      case "quarter":
        return React.createElement("div", {
          className: "govuk-grid-column-one-quarter " + (className || "")
        }, children);

      default:
        return React.createElement("div", {
          className: "govuk-grid-column-full " + (className || "")
        }, children);
    }
  }

  return renderGrid();
}

function HeaderLogo(_ref) {
  var title = _ref.title,
      _ref$withLogo = _ref.withLogo,
      withLogo = _ref$withLogo === void 0 ? true : _ref$withLogo;
  return React.createElement("div", {
    className: "idsk-header__logo"
  }, React.createElement("a", {
    className: "idsk-header__link idsk-header__link--homepage"
  }, withLogo && React.createElement("span", {
    className: "idsk-header__logotype"
  }, React.createElement(HeaderLogoSVG, null)), title && React.createElement("span", {
    className: "idsk-header__product-name"
  }, title)));
}

function HeaderLogoSVG() {
  return React.createElement("svg", {
    width: "888",
    height: "126",
    viewBox: "0 0 888 126",
    fill: "none",
    xmlns: "http://www.w3.org/2000/svg"
  }, React.createElement("path", {
    d: "M77.8007 96.3285C77.8007 102.59 76.2768 107.964 73.2291 112.453C70.1813 116.885 65.6651 120.265 59.6805 122.592C53.7512 124.864 46.5197 126 37.986 126C29.8402 126 22.8581 124.753 17.0397 122.26C11.2213 119.766 6.9267 116.664 4.15602 112.951C1.38534 109.183 0 105.416 0 101.648C0 99.1544 0.886618 97.0211 2.65985 95.248C4.43309 93.4749 6.67734 92.5884 9.39261 92.5884C11.7754 92.5884 13.604 93.1702 14.8786 94.3338C16.1531 95.4974 17.3722 97.1319 18.5359 99.2375C20.8632 103.282 23.6339 106.302 26.8479 108.297C30.1173 110.292 34.5504 111.289 40.1472 111.289C44.6911 111.289 48.4038 110.292 51.2853 108.297C54.2222 106.247 55.6907 103.92 55.6907 101.315C55.6907 97.3259 54.1668 94.4169 51.119 92.5884C48.1267 90.7599 43.1672 89.0145 36.2405 87.3522C28.4272 85.4129 22.0546 83.3905 17.1228 81.285C12.2464 79.124 8.33975 76.2982 5.40283 72.8074C2.46591 69.3166 0.997445 65.0224 0.997445 59.9248C0.997445 55.3813 2.35508 51.0871 5.07034 47.0422C7.78561 42.9974 11.7754 39.7836 17.0397 37.4011C22.3594 34.9631 28.7597 33.7441 36.2405 33.7441C42.1143 33.7441 47.3786 34.3536 52.0334 35.5726C56.7435 36.7916 60.6502 38.4261 63.7534 40.4763C66.9119 42.5264 69.2947 44.7982 70.9017 47.2916C72.5641 49.785 73.3953 52.223 73.3953 54.6055C73.3953 57.2098 72.5087 59.343 70.7355 61.0053C69.0176 62.6675 66.5517 63.4987 63.3377 63.4987C61.0104 63.4987 59.0155 62.8338 57.3531 61.504C55.7461 60.1741 53.8897 58.1794 51.784 55.5198C50.0662 53.3034 48.0436 51.5303 45.7162 50.2005C43.3889 48.8707 40.2303 48.2058 36.2405 48.2058C32.1399 48.2058 28.732 49.0923 26.0167 50.8654C23.3014 52.5831 21.9438 54.7441 21.9438 57.3483C21.9438 59.7309 22.9412 61.6979 24.9361 63.2493C26.931 64.7454 29.6186 65.9921 32.9988 66.9894C36.379 67.9868 41.0338 69.2058 46.963 70.6464C54.0006 72.3641 59.7359 74.4143 64.169 76.7968C68.6575 79.1794 72.0377 82.0053 74.3096 85.2744C76.637 88.4881 77.8007 92.1728 77.8007 96.3285Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M83 112.203V13.7968C83 9.2533 83.9974 5.81794 85.9923 3.49077C88.0426 1.16359 90.7856 0 94.2213 0C97.6569 0 100.428 1.16359 102.533 3.49077C104.639 5.76253 105.692 9.19789 105.692 13.7968V112.203C105.692 116.802 104.611 120.265 102.45 122.592C100.344 124.864 97.6015 126 94.2213 126C90.8964 126 88.1812 124.809 86.0755 122.426C84.0252 120.044 83 116.636 83 112.203Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M199.352 79.872C199.352 86.6319 198.299 92.8654 196.193 98.5726C194.088 104.28 191.04 109.183 187.05 113.284C183.06 117.384 178.295 120.542 172.753 122.759C167.212 124.92 160.978 126 154.051 126C147.18 126 141.001 124.892 135.515 122.675C130.029 120.459 125.264 117.301 121.219 113.201C117.229 109.045 114.181 104.169 112.075 98.5726C110.025 92.9208 109 86.6873 109 79.872C109 73.0013 110.053 66.7124 112.159 61.0053C114.264 55.2982 117.284 50.4222 121.219 46.3773C125.153 42.3325 129.919 39.2296 135.515 37.0686C141.112 34.8522 147.291 33.7441 154.051 33.7441C160.923 33.7441 167.157 34.8522 172.753 37.0686C178.35 39.285 183.143 42.4433 187.133 46.5435C191.123 50.6438 194.143 55.5198 196.193 61.1715C198.299 66.8232 199.352 73.0567 199.352 79.872ZM176.577 79.872C176.577 70.6187 174.527 63.4156 170.426 58.2625C166.381 53.1095 160.923 50.533 154.051 50.533C149.618 50.533 145.712 51.6966 142.331 54.0237C138.951 56.2955 136.347 59.6755 134.518 64.1636C132.689 68.6517 131.775 73.8879 131.775 79.872C131.775 85.8008 132.662 90.9815 134.435 95.4142C136.263 99.847 138.84 103.255 142.165 105.637C145.545 107.964 149.507 109.128 154.051 109.128C160.923 109.128 166.381 106.551 170.426 101.398C174.527 96.19 176.577 89.0145 176.577 79.872Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M217.691 46.4604L236.06 97.9908L255.843 44.2164C257.394 39.8945 258.918 36.847 260.414 35.0739C261.911 33.2454 264.183 32.3311 267.23 32.3311C270.112 32.3311 272.55 33.3008 274.545 35.2401C276.595 37.1794 277.62 39.4235 277.62 41.9723C277.62 42.9697 277.426 44.1332 277.039 45.4631C276.706 46.7929 276.29 48.0396 275.792 49.2032C275.348 50.3668 274.822 51.6966 274.212 53.1926L252.435 107.216C251.825 108.768 251.022 110.735 250.024 113.117C249.082 115.5 248.002 117.55 246.783 119.268C245.619 120.93 244.151 122.232 242.377 123.174C240.66 124.116 238.554 124.587 236.06 124.587C232.846 124.587 230.269 123.867 228.33 122.426C226.446 120.93 225.033 119.323 224.091 117.606C223.204 115.832 221.68 112.369 219.519 107.216L197.908 53.7744C197.409 52.4446 196.883 51.1148 196.329 49.785C195.83 48.4552 195.387 47.0976 194.999 45.7124C194.666 44.3272 194.5 43.1359 194.5 42.1385C194.5 40.5871 194.971 39.0633 195.913 37.5673C196.855 36.0158 198.157 34.7691 199.82 33.8272C201.482 32.8298 203.283 32.3311 205.223 32.3311C208.991 32.3311 211.567 33.4116 212.953 35.5726C214.393 37.7335 215.973 41.3628 217.691 46.4604Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M340.798 85.4406H296.412C296.467 90.5937 297.492 95.1372 299.487 99.0712C301.537 103.005 304.225 105.97 307.55 107.964C310.93 109.959 314.643 110.956 318.688 110.956C321.403 110.956 323.869 110.652 326.086 110.042C328.358 109.377 330.546 108.38 332.652 107.05C334.758 105.665 336.697 104.197 338.471 102.645C340.244 101.094 342.543 98.9881 345.37 96.3285C346.533 95.3311 348.196 94.8325 350.357 94.8325C352.684 94.8325 354.568 95.4697 356.009 96.7441C357.45 98.0185 358.17 99.8193 358.17 102.146C358.17 104.197 357.367 106.607 355.76 109.377C354.153 112.092 351.714 114.724 348.445 117.273C345.231 119.766 341.158 121.844 336.226 123.507C331.35 125.169 325.725 126 319.353 126C304.779 126 293.447 121.844 285.357 113.533C277.266 105.222 273.221 93.9459 273.221 79.7058C273.221 73.0013 274.218 66.7955 276.213 61.0884C278.208 55.3259 281.117 50.3945 284.941 46.2942C288.765 42.1939 293.475 39.0633 299.071 36.9024C304.668 34.686 310.875 33.5778 317.69 33.5778C326.557 33.5778 334.148 35.4617 340.465 39.2296C346.838 42.942 351.604 47.7625 354.762 53.6913C357.921 59.6201 359.5 65.6596 359.5 71.81C359.5 77.5172 357.865 81.2295 354.596 82.9472C351.326 84.6095 346.727 85.4406 340.798 85.4406ZM296.412 72.558H337.556C337.002 64.8008 334.896 59.0106 331.239 55.1873C327.637 51.3087 322.872 49.3694 316.942 49.3694C311.29 49.3694 306.635 51.3364 302.978 55.2705C299.376 59.1491 297.187 64.9116 296.412 72.558Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M385.403 46.2942V49.0369C389.392 43.7731 393.742 39.9222 398.453 37.4842C403.218 34.9908 408.676 33.7441 414.827 33.7441C420.812 33.7441 426.159 35.0462 430.87 37.6504C435.58 40.2546 439.098 43.9393 441.426 48.7045C442.922 51.4749 443.892 54.467 444.335 57.6807C444.778 60.8945 445 64.9947 445 69.9815V112.203C445 116.747 443.947 120.182 441.841 122.509C439.791 124.836 437.104 126 433.779 126C430.398 126 427.656 124.809 425.55 122.426C423.444 120.044 422.391 116.636 422.391 112.203V74.3865C422.391 66.9063 421.338 61.1992 419.233 57.2652C417.182 53.2757 413.054 51.281 406.848 51.281C402.803 51.281 399.118 52.5 395.793 54.938C392.468 57.3206 390.03 60.6174 388.478 64.8285C387.37 68.2084 386.816 74.5251 386.816 83.7784V112.203C386.816 116.802 385.735 120.265 383.574 122.592C381.468 124.864 378.725 126 375.345 126C372.076 126 369.388 124.809 367.282 122.426C365.177 120.044 364.124 116.636 364.124 112.203V46.6266C364.124 42.3047 365.066 39.091 366.95 36.9855C368.834 34.8245 371.411 33.7441 374.68 33.7441C376.675 33.7441 378.476 34.215 380.083 35.157C381.69 36.0989 382.964 37.5119 383.906 39.3958C384.904 41.2797 385.403 43.5792 385.403 46.2942Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M526.301 96.3285C526.301 102.59 524.777 107.964 521.729 112.453C518.681 116.885 514.165 120.265 508.18 122.592C502.251 124.864 495.02 126 486.486 126C478.34 126 471.358 124.753 465.54 122.26C459.721 119.766 455.427 116.664 452.656 112.951C449.885 109.183 448.5 105.416 448.5 101.648C448.5 99.1544 449.387 97.0211 451.16 95.248C452.933 93.4749 455.177 92.5884 457.893 92.5884C460.275 92.5884 462.104 93.1702 463.379 94.3338C464.653 95.4974 465.872 97.1319 467.036 99.2375C469.363 103.282 472.134 106.302 475.348 108.297C478.617 110.292 483.05 111.289 488.647 111.289C493.191 111.289 496.904 110.292 499.785 108.297C502.722 106.247 504.191 103.92 504.191 101.315C504.191 97.3259 502.667 94.4169 499.619 92.5884C496.627 90.7599 491.667 89.0145 484.741 87.3522C476.927 85.4129 470.555 83.3905 465.623 81.285C460.746 79.124 456.84 76.2982 453.903 72.8074C450.966 69.3166 449.497 65.0224 449.497 59.9248C449.497 55.3813 450.855 51.0871 453.57 47.0422C456.286 42.9974 460.275 39.7836 465.54 37.4011C470.859 34.9631 477.26 33.7441 484.741 33.7441C490.614 33.7441 495.879 34.3536 500.533 35.5726C505.244 36.7916 509.15 38.4261 512.253 40.4763C515.412 42.5264 517.795 44.7982 519.402 47.2916C521.064 49.785 521.895 52.223 521.895 54.6055C521.895 57.2098 521.009 59.343 519.235 61.0053C517.518 62.6675 515.052 63.4987 511.838 63.4987C509.51 63.4987 507.515 62.8338 505.853 61.504C504.246 60.1741 502.39 58.1794 500.284 55.5198C498.566 53.3034 496.544 51.5303 494.216 50.2005C491.889 48.8707 488.73 48.2058 484.741 48.2058C480.64 48.2058 477.232 49.0923 474.517 50.8654C471.801 52.5831 470.444 54.7441 470.444 57.3483C470.444 59.7309 471.441 61.6979 473.436 63.2493C475.431 64.7454 478.119 65.9921 481.499 66.9894C484.879 67.9868 489.534 69.2058 495.463 70.6464C502.501 72.3641 508.236 74.4143 512.669 76.7968C517.157 79.1794 520.538 82.0053 522.81 85.2744C525.137 88.4881 526.301 92.1728 526.301 96.3285Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M585.309 115.611L565.277 82.6979L552.976 94.3338V112.369C552.976 116.747 551.812 120.127 549.484 122.509C547.213 124.836 544.58 126 541.588 126C538.097 126 535.354 124.836 533.359 122.509C531.364 120.182 530.367 116.747 530.367 112.203V15.4591C530.367 10.4169 531.337 6.59367 533.276 3.98945C535.216 1.32982 537.986 0 541.588 0C545.079 0 547.85 1.19129 549.9 3.57388C551.95 5.95646 552.976 9.47493 552.976 14.1293V69.1504L578.494 42.3879C581.652 39.0633 584.063 36.7916 585.725 35.5726C587.387 34.3536 589.41 33.7441 591.793 33.7441C594.619 33.7441 596.974 34.6583 598.858 36.4868C600.742 38.2599 601.684 40.504 601.684 43.219C601.684 46.5435 598.609 50.9763 592.458 56.5172L580.405 67.5712L603.679 104.141C605.397 106.856 606.616 108.934 607.336 110.375C608.112 111.76 608.5 113.09 608.5 114.364C608.5 117.966 607.503 120.819 605.508 122.925C603.568 124.975 600.991 126 597.777 126C595.007 126 592.873 125.252 591.377 123.756C589.881 122.26 587.858 119.545 585.309 115.611Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M695.5 79.872C695.5 86.6319 694.447 92.8654 692.341 98.5726C690.236 104.28 687.188 109.183 683.198 113.284C679.208 117.384 674.443 120.542 668.901 122.759C663.36 124.92 657.126 126 650.199 126C643.328 126 637.149 124.892 631.664 122.675C626.178 120.459 621.412 117.301 617.367 113.201C613.377 109.045 610.329 104.169 608.224 98.5726C606.173 92.9208 605.148 86.6873 605.148 79.872C605.148 73.0013 606.201 66.7124 608.307 61.0053C610.412 55.2982 613.432 50.4222 617.367 46.3773C621.301 42.3325 626.067 39.2296 631.664 37.0686C637.26 34.8522 643.439 33.7441 650.199 33.7441C657.071 33.7441 663.305 34.8522 668.901 37.0686C674.498 39.285 679.292 42.4433 683.281 46.5435C687.271 50.6438 690.291 55.5198 692.341 61.1715C694.447 66.8232 695.5 73.0567 695.5 79.872ZM672.725 79.872C672.725 70.6187 670.675 63.4156 666.574 58.2625C662.529 53.1095 657.071 50.533 650.199 50.533C645.766 50.533 641.86 51.6966 638.479 54.0237C635.099 56.2955 632.495 59.6755 630.666 64.1636C628.837 68.6517 627.923 73.8879 627.923 79.872C627.923 85.8008 628.81 90.9815 630.583 95.4142C632.412 99.847 634.988 103.255 638.313 105.637C641.693 107.964 645.655 109.128 650.199 109.128C657.071 109.128 662.529 106.551 666.574 101.398C670.675 96.19 672.725 89.0145 672.725 79.872Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M707.134 126C703.699 126 700.734 124.892 698.24 122.675C695.747 120.459 694.5 117.356 694.5 113.367C694.5 109.987 695.691 107.078 698.074 104.64C700.457 102.202 703.394 100.983 706.885 100.983C710.376 100.983 713.341 102.202 715.779 104.64C718.272 107.022 719.519 109.931 719.519 113.367C719.519 117.301 718.272 120.404 715.779 122.675C713.341 124.892 710.459 126 707.134 126Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M804 96.3285C804 102.59 802.476 107.964 799.428 112.453C796.381 116.885 791.864 120.265 785.88 122.592C779.951 124.864 772.719 126 764.185 126C756.04 126 749.057 124.753 743.239 122.26C737.421 119.766 733.126 116.664 730.355 112.951C727.585 109.183 726.199 105.416 726.199 101.648C726.199 99.1544 727.086 97.0211 728.859 95.248C730.632 93.4749 732.877 92.5884 735.592 92.5884C737.975 92.5884 739.803 93.1702 741.078 94.3338C742.352 95.4974 743.571 97.1319 744.735 99.2375C747.063 103.282 749.833 106.302 753.047 108.297C756.317 110.292 760.75 111.289 766.346 111.289C770.89 111.289 774.603 110.292 777.485 108.297C780.422 106.247 781.89 103.92 781.89 101.315C781.89 97.3259 780.366 94.4169 777.318 92.5884C774.326 90.7599 769.367 89.0145 762.44 87.3522C754.627 85.4129 748.254 83.3905 743.322 81.285C738.446 79.124 734.539 76.2982 731.602 72.8074C728.665 69.3166 727.197 65.0224 727.197 59.9248C727.197 55.3813 728.554 51.0871 731.27 47.0422C733.985 42.9974 737.975 39.7836 743.239 37.4011C748.559 34.9631 754.959 33.7441 762.44 33.7441C768.314 33.7441 773.578 34.3536 778.233 35.5726C782.943 36.7916 786.85 38.4261 789.953 40.4763C793.111 42.5264 795.494 44.7982 797.101 47.2916C798.763 49.785 799.595 52.223 799.595 54.6055C799.595 57.2098 798.708 59.343 796.935 61.0053C795.217 62.6675 792.751 63.4987 789.537 63.4987C787.21 63.4987 785.215 62.8338 783.552 61.504C781.945 60.1741 780.089 58.1794 777.983 55.5198C776.266 53.3034 774.243 51.5303 771.916 50.2005C769.588 48.8707 766.43 48.2058 762.44 48.2058C758.339 48.2058 754.931 49.0923 752.216 50.8654C749.501 52.5831 748.143 54.7441 748.143 57.3483C748.143 59.7309 749.141 61.6979 751.135 63.2493C753.13 64.7454 755.818 65.9921 759.198 66.9894C762.578 67.9868 767.233 69.2058 773.162 70.6464C780.2 72.3641 785.935 74.4143 790.368 76.7968C794.857 79.1794 798.237 82.0053 800.509 85.2744C802.836 88.4881 804 92.1728 804 96.3285Z",
    fill: "white"
  }), React.createElement("path", {
    d: "M864.443 115.611L844.411 82.6979L832.109 94.3338V112.369C832.109 116.747 830.945 120.127 828.618 122.509C826.346 124.836 823.714 126 820.721 126C817.23 126 814.487 124.836 812.492 122.509C810.497 120.182 809.5 116.747 809.5 112.203V15.4591C809.5 10.4169 810.47 6.59367 812.409 3.98945C814.349 1.32982 817.119 0 820.721 0C824.212 0 826.983 1.19129 829.033 3.57388C831.084 5.95646 832.109 9.47493 832.109 14.1293V69.1504L857.627 42.3879C860.785 39.0633 863.196 36.7916 864.858 35.5726C866.521 34.3536 868.543 33.7441 870.926 33.7441C873.752 33.7441 876.107 34.6583 877.991 36.4868C879.875 38.2599 880.817 40.504 880.817 43.219C880.817 46.5435 877.742 50.9763 871.591 56.5172L859.538 67.5712L882.812 104.141C884.53 106.856 885.749 108.934 886.469 110.375C887.245 111.76 887.633 113.09 887.633 114.364C887.633 117.966 886.636 120.819 884.641 122.925C882.701 124.975 880.125 126 876.911 126C874.14 126 872.007 125.252 870.51 123.756C869.014 122.26 866.992 119.545 864.443 115.611Z",
    fill: "white"
  }));
}

function HeaderNavigation(_ref) {
  var service = _ref.service,
      navigation = _ref.navigation;
  return React.createElement("div", {
    className: "idsk-header__content"
  }, service && React.createElement("a", {
    className: "idsk-header__link idsk-header__link--service-name"
  }, service), navigation && React.createElement("button", {
    type: "button",
    className: "idsk-header__menu-button govuk-js-header-toggle",
    "aria-controls": "navigation",
    "aria-label": "Show or hide Top Level Navigation"
  }, "Menu"), navigation && React.createElement("nav", null, React.createElement("ul", {
    id: "navigation",
    className: "idsk-header__navigation ",
    "aria-label": "Top Level Navigation"
  }, navigation.map(function (nav) {
    return React.createElement("li", {
      key: nav.item,
      className: "idsk-header__navigation-item " + (nav.active ? "idsk-header__navigation-item--active" : "")
    }, React.createElement("a", {
      className: "idsk-header__link",
      href: nav.href
    }, nav.item));
  }))));
}

function HeaderUser(_ref) {
  var onLogout = _ref.onLogout,
      user = _ref.user;
  return React.createElement("div", {
    className: "idsk-header__user govuk-grid-column-one-quarter idsk-header__user--end"
  }, React.createElement(UserIcon, null), React.createElement("div", {
    className: "idsk--header__user-name"
  }, user && React.createElement("span", null, user), React.createElement("div", null, React.createElement("a", {
    onClick: onLogout,
    href: "#",
    className: "idsk--header__user-logout"
  }, "Odhl\xE1si\u0165 sa"))));
}

function UserIcon() {
  return React.createElement("svg", {
    className: "idsk-header__user-icon",
    viewBox: "0 0 28 28"
  }, React.createElement("path", {
    d: "M5 21.9C5 19.8 6.8 18 9 18h10c2.2 0 4 1.8 4 3.9 1.9-2.1 3-4.9 3-7.9 0-6.6-5.4-12-12-12S2 7.4 2 14c0 3 1.1 5.8 3 7.9zm9 6.1C6.3 28 0 21.7 0 14S6.3 0 14 0s14 6.3 14 14-6.3 14-14 14zm0-12c-2.8 0-5-2.2-5-5s2.2-5 5-5 5 2.2 5 5-2.2 5-5 5z",
    fillRule: "evenodd",
    clipRule: "evenodd"
  }));
}

function Header(_ref) {
  var className = _ref.className,
      _ref$fullWidth = _ref.fullWidth,
      fullWidth = _ref$fullWidth === void 0 ? false : _ref$fullWidth,
      navigation = _ref.navigation,
      service = _ref.service,
      title = _ref.title,
      user = _ref.user,
      withLogo = _ref.withLogo,
      onLogout = _ref.onLogout;
  return React.createElement("header", {
    className: "idsk-header " + (className || ""),
    role: "banner",
    "data-module": "idsk-header"
  }, React.createElement("div", {
    className: "idsk-header__container " + (fullWidth ? "govuk-header__container--full-width" : "govuk-width-container") + " " + (onLogout ? "govuk-header__container--with-user" : "")
  }, React.createElement(HeaderLogo, {
    title: title,
    withLogo: withLogo
  }), React.createElement(HeaderNavigation, {
    service: service,
    navigation: navigation
  }), onLogout && React.createElement(HeaderUser, {
    onLogout: onLogout,
    user: user
  })));
}

function Typography(_ref) {
  var children = _ref.children,
      className = _ref.className,
      _ref$href = _ref.href,
      href = _ref$href === void 0 ? "#" : _ref$href,
      onClick = _ref.onClick,
      _ref$type = _ref.type,
      type = _ref$type === void 0 ? "p" : _ref$type,
      _ref$size = _ref.size,
      size = _ref$size === void 0 ? "m" : _ref$size;

  function renderText() {
    switch (type) {
      case "h1":
        return React.createElement(TypographyHeaderOne, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });

      case "h2":
        return React.createElement(TypographyHeaderTwo, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });

      case "h3":
        return React.createElement(TypographyHeaderThree, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });

      case "h4":
        return React.createElement(TypographyHeaderFour, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });

      case "a":
        return React.createElement(TypographyAnchor, {
          children: children,
          className: className,
          href: href,
          onClick: onClick,
          size: size
        });

      case "p":
        return React.createElement(TypographyParagraph, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });

      default:
        return React.createElement(TypographyParagraph, {
          children: children,
          className: className,
          onClick: onClick,
          size: size
        });
    }
  }

  return renderText();
}

function TypographyHeaderOne(_ref2) {
  var children = _ref2.children,
      className = _ref2.className,
      onClick = _ref2.onClick,
      size = _ref2.size;
  return React.createElement("h1", {
    onClick: onClick,
    style: onClick && {
      cursor: "pointer"
    },
    className: "govuk-heading-" + size + " " + (className || "")
  }, children);
}

function TypographyHeaderTwo(_ref3) {
  var children = _ref3.children,
      className = _ref3.className,
      onClick = _ref3.onClick,
      size = _ref3.size;
  return React.createElement("h2", {
    onClick: onClick,
    style: onClick && {
      cursor: "pointer"
    },
    className: "govuk-heading-" + size + " " + (className || "")
  }, children);
}

function TypographyHeaderThree(_ref4) {
  var children = _ref4.children,
      className = _ref4.className,
      onClick = _ref4.onClick,
      size = _ref4.size;
  return React.createElement("h3", {
    onClick: onClick,
    style: onClick && {
      cursor: "pointer"
    },
    className: "govuk-heading-" + size + " " + (className || "")
  }, children);
}

function TypographyHeaderFour(_ref5) {
  var children = _ref5.children,
      className = _ref5.className,
      onClick = _ref5.onClick,
      size = _ref5.size;
  return React.createElement("h4", {
    onClick: onClick,
    style: onClick && {
      cursor: "pointer"
    },
    className: "govuk-heading-" + size + " " + (className || "")
  }, children);
}

function TypographyAnchor(_ref6) {
  var children = _ref6.children,
      className = _ref6.className,
      href = _ref6.href,
      onClick = _ref6.onClick,
      size = _ref6.size;
  return React.createElement("a", {
    onClick: onClick,
    style: onClick && {
      cursor: "pointer"
    },
    className: "govuk-body-" + size + " govuk-link " + (className || ""),
    href: href
  }, children);
}

function TypographyParagraph(_ref7) {
  var children = _ref7.children,
      className = _ref7.className,
      onClick = _ref7.onClick,
      size = _ref7.size;
  return React.createElement("p", {
    style: onClick && {
      cursor: "pointer"
    },
    onClick: onClick,
    className: "govuk-body-" + size + " " + (className || "")
  }, children);
}

function Loading() {
  return React.createElement(Typography, null, "Na\u010D\xEDtavam ...");
}

function TableCell(_ref) {
  var isPrimary = _ref.isPrimary,
      value = _ref.value,
      _ref$align = _ref.align,
      align = _ref$align === void 0 ? "left" : _ref$align;
  return isPrimary ? React.createElement("th", {
    scope: "row",
    className: "govuk-table__header " + (align === "right" ? "govuk-table__cell--numeric" : "")
  }, value) : React.createElement("td", {
    className: "govuk-table__cell " + (align === "right" ? "govuk-table__cell--numeric" : "")
  }, value);
}

function Table(_ref) {
  var columns = _ref.columns,
      data = _ref.data,
      primaryKey = _ref.primaryKey,
      title = _ref.title,
      _ref$withHeader = _ref.withHeader,
      withHeader = _ref$withHeader === void 0 ? true : _ref$withHeader;
  return React.createElement("table", {
    className: "govuk-table"
  }, title && React.createElement("caption", {
    className: "govuk-table__caption govuk-heading-m"
  }, title), withHeader && React.createElement("thead", {
    className: "govuk-table__head"
  }, React.createElement("tr", {
    className: "govuk-table__row"
  }, columns.map(function (col) {
    return React.createElement("th", {
      key: "header-" + col.key,
      scope: "col",
      className: "govuk-table__header " + (col.align === "right" ? "govuk-table__header govuk-table__header--numeric" : "")
    }, col.label);
  }))), React.createElement("tbody", {
    className: "govuk-table__body"
  }, data.length > 0 && data.map(function (it, dataIndex) {
    return React.createElement("tr", {
      key: primaryKey + "-" + dataIndex,
      className: "govuk-table__row"
    }, columns.map(function (col, colIndex) {
      var isPrimary = it[primaryKey] === it[col.key];
      var valueToRender = col.render ? col.render(it) : it[col.key];
      return React.createElement(TableCell, {
        align: col.align,
        key: colIndex + "-" + dataIndex,
        isPrimary: isPrimary,
        value: valueToRender
      });
    }));
  })));
}

function TabPanel(_ref) {
  var children = _ref.children,
      index = _ref.index,
      tabIndex = _ref.tabIndex,
      title = _ref.title;
  return React.createElement("section", {
    className: "govuk-tabs__panel " + (index !== tabIndex ? "govuk-tabs__panel--hidden" : ""),
    role: "tabpanel",
    id: index + "-tab"
  }, title && React.createElement("h2", {
    className: "govuk-heading-l"
  }, title), children);
}

function Tabs(_ref) {
  var tabIndex = _ref.tabIndex,
      children = _ref.children,
      onChange = _ref.onChange,
      tabs = _ref.tabs,
      title = _ref.title;

  function onTabChange(newValue) {
    onChange(newValue);
  }

  return React.createElement("div", {
    className: "govuk-tabs",
    "data-module": "govuk-tabs"
  }, title && React.createElement("h2", {
    className: "govuk-tabs__title"
  }, title), React.createElement("ul", {
    className: "govuk-tabs__list",
    role: "tablist"
  }, tabs.map(function (it, index) {
    return React.createElement("li", {
      key: it.id,
      role: "presentation",
      className: "govuk-tabs__list-item " + (index === tabIndex ? "govuk-tabs__list-item--selected" : ""),
      onClick: function onClick() {
        return onTabChange(index);
      }
    }, React.createElement("a", {
      role: "tab",
      className: "govuk-tabs__tab",
      href: "#" + index + "-tab",
      tabIndex: index === tabIndex ? 0 : -1
    }, it.tab));
  })), children);
}

function CheckboxInput(_ref) {
  var name = _ref.name,
      className = _ref.className,
      defaultValue = _ref.defaultValue,
      disabled = _ref.disabled,
      error = _ref.error,
      errorClassName = _ref.errorClassName,
      helperClassName = _ref.helperClassName,
      helperText = _ref.helperText,
      label = _ref.label,
      labelClassName = _ref.labelClassName,
      value = _ref.value,
      _ref$type = _ref.type,
      type = _ref$type === void 0 ? "checkbox" : _ref$type,
      onChange = _ref.onChange,
      checked = _ref.checked,
      _ref$small = _ref.small,
      small = _ref$small === void 0 ? false : _ref$small;
  return React.createElement("div", {
    className: "govuk-form-group " + (error ? "govuk-form-group--error" : "")
  }, helperText && React.createElement("span", {
    id: name + "-hint\"",
    className: "govuk-hint " + (helperClassName || "")
  }, helperText), error && React.createElement("span", {
    id: name + "-error",
    className: "govuk-error-message " + (errorClassName || "")
  }, React.createElement("span", {
    className: "govuk-visually-hidden"
  }, "Error:"), " ", error), React.createElement("div", {
    className: "" + (small ? "govuk-checkboxes--small" : "govuk-checkboxes")
  }, React.createElement("div", {
    className: "govuk-checkboxes__item"
  }, React.createElement("input", {
    checked: checked,
    onChange: onChange,
    disabled: disabled,
    className: "govuk-checkboxes__input " + (className || ""),
    id: name,
    name: name,
    type: type,
    value: value,
    defaultValue: defaultValue
  }), React.createElement("label", {
    className: "govuk-label govuk-checkboxes__label " + (labelClassName || ""),
    htmlFor: name
  }, label))));
}

function SelectInput(_ref) {
  var name = _ref.name,
      options = _ref.options,
      className = _ref.className,
      defaultValue = _ref.defaultValue,
      disabled = _ref.disabled,
      error = _ref.error,
      errorClassName = _ref.errorClassName,
      helperClassName = _ref.helperClassName,
      helperText = _ref.helperText,
      label = _ref.label,
      labelClassName = _ref.labelClassName,
      value = _ref.value,
      onChange = _ref.onChange;
  return React.createElement("div", {
    className: "govuk-form-group " + (error ? "govuk-form-group--error" : "")
  }, label && React.createElement("label", {
    className: "govuk-label " + (labelClassName || ""),
    htmlFor: name
  }, label), helperText && React.createElement("span", {
    id: name + "-hint\"",
    className: "govuk-hint " + (helperClassName || "")
  }, helperText), error && React.createElement("span", {
    id: name + "-error",
    className: "govuk-error-message " + (errorClassName || "")
  }, React.createElement("span", {
    className: "govuk-visually-hidden"
  }, "Error:"), " ", error), React.createElement("select", {
    className: "govuk-select " + (className || ""),
    id: name,
    disabled: disabled,
    name: name,
    defaultValue: defaultValue,
    value: value,
    onChange: onChange
  }, options.map(function (option) {
    return React.createElement("option", {
      key: option.value,
      value: option.value
    }, option.label);
  })));
}

function TextInput(_ref) {
  var name = _ref.name,
      className = _ref.className,
      defaultValue = _ref.defaultValue,
      disabled = _ref.disabled,
      error = _ref.error,
      errorClassName = _ref.errorClassName,
      helperClassName = _ref.helperClassName,
      fullWidth = _ref.fullWidth,
      helperText = _ref.helperText,
      label = _ref.label,
      labelClassName = _ref.labelClassName,
      value = _ref.value,
      _ref$type = _ref.type,
      type = _ref$type === void 0 ? "text" : _ref$type,
      onChange = _ref.onChange,
      _ref$width = _ref.width,
      width = _ref$width === void 0 ? 30 : _ref$width;
  return React.createElement("div", {
    className: "govuk-form-group " + (error ? "govuk-form-group--error" : "")
  }, label && React.createElement("label", {
    className: "govuk-label " + (labelClassName || ""),
    htmlFor: name
  }, label), helperText && React.createElement("span", {
    id: name + "-hint\"",
    className: "govuk-hint " + (helperClassName || "")
  }, helperText), error && React.createElement("span", {
    id: name + "-error",
    className: "govuk-error-message " + (errorClassName || "")
  }, React.createElement("span", {
    className: "govuk-visually-hidden"
  }, "Error:"), " ", error), React.createElement("input", {
    "aria-describedby": name + "-hint",
    className: "govuk-input " + (error ? "govuk-input--error" : "") + " " + (!fullWidth && width ? "govuk-input--width-" + width : "") + " " + (className || ""),
    disabled: disabled,
    id: name,
    name: name,
    value: value,
    defaultValue: defaultValue,
    type: type,
    onChange: onChange
  }));
}

idsk__default.initAll();

exports.Alert = Alert;
exports.Button = Button;
exports.CheckboxInput = CheckboxInput;
exports.Container = Container;
exports.Footer = Footer;
exports.Grid = Grid;
exports.Header = Header;
exports.Loading = Loading;
exports.SelectInput = SelectInput;
exports.TabPanel = TabPanel;
exports.Table = Table;
exports.Tabs = Tabs;
exports.TextInput = TextInput;
exports.Typography = Typography;
